Lib/test/test_syntax.py

```diff
@@ -926,6 +926,114 @@
    Traceback (most recent call last):
    SyntaxError: invalid syntax
 
+Specialized indentation errors:
```

Comment:
I think you're missing a test for `while`, `case` and `match` statements.

Reply:
Thanks for the catch, will add those soon